fvm/accountV2Migration/AccountV2Migration.cdc

```diff
@@ -0,0 +1,102 @@
+access(all)
+contract AccountV2Migration {
+
+    access(all)
+    enum StorageFormat: UInt8 {
+        access(all)
+        case Unknown
+
+        access(all)
+        case V1
+
+        access(all)
+        case V2
+    }
+
+    access(all)
+    event Migrated(
+        addressStartIndex: UInt64,
+        count: UInt64
+    )
+
+    access(all)
+    resource Admin {
+        access(all)
+        fun setEnabled(_ isEnabled: Bool) {
+            AccountV2Migration.isEnabled = isEnabled
+        }
+
+        access(all)
+        fun setNextAddressStartIndex(_ nextAddressStartIndex: UInt64) {
+            AccountV2Migration.nextAddressStartIndex = nextAddressStartIndex
+        }
+
+        access(all)
+        fun setBatchSize(_ batchSize: UInt64) {
+            AccountV2Migration.batchSize = batchSize
+        }
```

Comment:
Do we want to have a minimum `batchSize` that is larger than 0?  

It seems migration is effectively disabled when `batchSize` is 0.  Maybe forcing caller to explicitly use `setEnabled(false)` would reduce mistakes.

Reply:
Good point, the enabled flag isn't really needed, we can disable via the batch size. Changed it in [f86dd29](https://github.com/onflow/flow-go/pull/6761/commits/f86dd292a2b788d198b31c73c7784ed9ca74ae56)